scripts/setupvars/setupvars.bat

```diff
@@ -27,25 +26,14 @@ if exist "%INTEL_OPENVINO_DIR%\extras\opencv\setupvars.bat" (
 call "%INTEL_OPENVINO_DIR%\extras\opencv\setupvars.bat"
 goto :opencv_done
 )
-if exist "%INTEL_OPENVINO_DIR%\opencv\setupvars.bat" (
```

Comment:
Can we align behavior between win and lin by calling ocv setupvars from extras folder if it is exists? CC @alalek

Reply:
We provide two old and new options to provide smooth migration before packaging layout is upgraded.
It can be removed now.
Keep `extras\opencv` only.